netbox/project-static/styles/overrides/_tabler.scss

```diff
@@ -66,6 +66,19 @@ table th.orderable a {
   background: var(--tblr-bg-surface-secondary)!important;
 }
 
+// Altering dashboard card colors
+.grid-stack {
+  .card-header {
+    // Default color of card header changes depending on theme
+    &.bg-default {
+      background: var(--tblr-bg-surface-secondary)!important;
+    }
+    a {
+      color: inherit!important;
+    }
+  }
+}
+
```

Comment:
Let's move these to a new file, `overrides/_gridstack.css`, and include it in `netbox.scss`, as these classes come from the [gridstack.js](https://gridstackjs.com/) package rather than Tabler.

Reply:
fixed.